src/index.tsx

```diff
@@ -185,7 +185,7 @@ function Dashboard() {
             <AppBar position="sticky">
                 <Toolbar>
                     {shouldRenderBackButton && (
-                        <IconButton onClick={() => window.close()} edge="start" color="inherit" aria-label="menu">
+                        <IconButton onClick={() => window.close()} edge="start" color="inherit" aria-label="back">
```

Comment:
(actually, aria-label will be read out by the screen reader. if you want to use a aria-label, you also need to do the i18n here.)